node/network/statement-distribution/src/tests.rs

```diff
@@ -2004,19 +2007,31 @@ fn handle_multiple_seconded_statements() {
 				our_neighbors_y: HashMap::new(),
 			};
 
-			// This is relying on the fact that statement distribution
-			// just extracts the peer IDs from this struct and does nothing else
-			// with it.
+			// Create a topology to ensure that we send messages not to `peer_a`/`peer_b`
 			for (i, peer) in lucky_peers.iter().enumerate() {
 				let authority_id = AuthorityPair::generate().0.public();
-				t.our_neighbors_x.insert(
+				t.our_neighbors_y.insert(
```

Comment:
Could you explain this subtle change?

Reply:
The idea of this test was to send updates to all other peers but not to `peer_a`/`peer_b`, that's why I've just moved them to the different axis.

Reply:
Alright, makes sense